api.yaml

```diff
@@ -1062,12 +1062,18 @@ components:
           description: Specify the maximum amount of tickets that can be added to a single order. The absence of this field will mean that the product doesn't have such limits.
         required_order_data:
           type: array
-          description: List of additional fields that need to be provided while creating a reservation. Each field will contain a piece of information about the whole order.
+          description: |
+            List of _additional_, order-level details that must be included when creating a reservation.  
+            Tiqets _always_ collects the `name`, `email` and `phone number` of the main booker which is why there are no values for that in this list.  
```

Comment:
BTW @mennobb we're sending fake data by default and there is no way for the supplier to request the real data.

Reply:
Yes, I'm aware of that but chose to leave it out of the documentation.
Thank you for noticing though!